processing/src/main/java/org/apache/druid/segment/incremental/OnheapIncrementalIndex.java

```diff
@@ -435,6 +490,18 @@ public boolean isNull(int rowOffset, int aggOffset)
     }
   }
 
+  private Object getCombinedMetric(AggregatorFactory[] metrics, Aggregator[] aggs, int aggOffset)
```

Comment:
this seems worthy of javadoc/comments

Reply:
This method is renamed to getMetricHelper and does have javadoc now.